Testscripts/Windows/SETUP-RemoveIsoFromDvd.ps1

```diff
@@ -57,10 +57,36 @@ function Main {
     }
 
     if (-not $?) {
-        "Error: Unable to remove the .iso from the DVD!"
+        Write-LogErr "Unable to remove the .iso from the DVD!"
         return $False
     }
 
+# Get Hyper-V VHD path
+$obj = Get-WmiObject -ComputerName $hvServer -Namespace "root\virtualization\v2" -Class "MsVM_VirtualSystemManagementServiceSettingData"
+$defaultVhdPath = $obj.DefaultVirtualHardDiskPath
+if (-not $defaultVhdPath) {
+    Write-LogErr "Unable to determine VhdDefaultPath on Hyper-V server ${hvServer}"
+    $error[0].Exception
```

Comment:
what is this for?

Reply:
All these lines are used to find the default location of vm vhdpath, since in the LIS-Mount-CD, iso put on the save vhd file location.
 $isoPath = $defaultVhdPath + "${vmName}_CDtest.iso", when remove iso file, also need to find the path.

For $error[0].exception is used to output the error message if cannot find the default vhdx. It should put together with "Write-LogErr", but there are multiple $error in LIS-Mount-CD.ps1, e.g. line 83 in LIS-Mount-CD.ps1, so I keep there.